src/components/user/Context.tsx

```diff
@@ -49,24 +49,39 @@ const UserContextProvider: React.FC<UserContextProviderProps> = (
     _userBadgesImported: UserBadge[],
     _communityBadges: CommunityBadge[]
   ) => {
-    const _userBadgesToImport = _userBadges.filter(({ assetCode }) => {
-      if (!assetCode) return false;
-      const communityIncludesBadge = _communityBadges.some(
-        ({ assetCode: communityBadgeAssetCode }) => {
-          return (
-            communityBadgeAssetCode.toLocaleLowerCase() ===
-            assetCode.toLocaleLowerCase()
-          );
-        }
-      );
-      const userBadgesImportedIncludesBadge = _userBadgesImported.some(
-        ({ assetCode: importedBadgeAssetCode }) =>
-          !!importedBadgeAssetCode &&
-          assetCode.toLocaleLowerCase() ===
-            importedBadgeAssetCode.toLocaleLowerCase()
-      );
-      return communityIncludesBadge && !userBadgesImportedIncludesBadge;
-    });
+    // All blockful badges are opened to import, 
+    // so we create this array to consider all blockfulBadges as badges to be imported..
+    const blockfulBadges: UserBadge[] = _communityBadges
+      .filter(({ questName }) => questName === "Blockful-Badges")
```

Comment:
```suggestion
      .filter(({ questName }) => questName === BLOCKFUL_BADGES_KEY)
```

would love to see this alongside with the definition of a `const BLOCKFUL_BADGES_KEY = "Blockful-Badges"`